.github/workflows/release-drafter.yml

```diff
@@ -2,15 +2,20 @@ name: Release Drafter
 
 on:
   push:
-    branches: [main]
+    tags:
+      - 'v*'
 
 jobs:
   update_release_draft:
+    if: github.event.base_ref == 'refs/heads/main'
```

Comment:
Hmm 🤔 what does `base_ref` mean in context of a workflow executed **for a tag**?

Reply:
Hm, I am not quite sure. Tag is related to a commit, not to branch, so it is somehow confusing. I followed an advice from https://github.community/t/run-workflow-on-push-tag-on-specific-branch/17519 which did work for my private repo - it did recognise branch name. Looks like base_ref contained information about branch connected with the commit. But now when I think about it and do some reading - it seems that it does not work always and might not work. Let me remove this line in upcomming commit. We should go without this check.

Reply:
this `if` condition has been removed in the newest commit

Reply:
Huh actually... let's assume for a moment the (likely) scenario when someone pushes a tag for a commit that is NOT a part of `master` history.

How does the behavior differ for such case with and without `if: github.event.base_ref == 'refs/heads/main'`? 

My suspicion is that this `if` might protect us from accidentally releasing a commit that is not a part of master 🤔

Reply:
yes, that was the reason I added this `if` - however: it worked well on my private repo but I saw some discussions on the web, that github.event.base_ref might be null when tag is the source of deploying workflow. Let me check that some deeper on my repo - I will try to come back with some solution.

Reply:
without this `if`- such github release will be created also for 'v*' tags related to commits, that were not merged to main branch

Reply:
Theoreticaly, if the `if` doesn't work as expected... then you could also add a check that `git rev-parse master` is equal to `git rev-parse HEAD`, directly as a script in the pipeline (and fail if the two hashes are NOT equal)

Reply:
Ok, I tested it more on my private repo - it showed up that `if: github.event.base_ref == 'refs/heads/main'` works well both for Merge commits and without them (we do not have merge commits in ash now). The event for which there are context properties available - is **push** - and it works well for pushing tags - github is able to recognize base_ref  related to this tag push - which is the branch. So I will just paste this if statement back.

Reply:
`if` statement added (after tests)